handlers/vk/vk.go

```diff
@@ -417,6 +445,58 @@ func (h *handler) SendMsg(ctx context.Context, msg courier.Msg) (courier.MsgStat
 	return status, nil
 }
 
+// ToJSON returns the JSON encoding of Keyboard.
+func ToJSON(keyboard Keyboard) string {
+	b, _ := json.Marshal(keyboard)
+	return string(b)
+}
+
+//keyboard
+func buildQuickRepliesParams(msg courier.Msg) Keyboard {
+
+	var keyboard Keyboard
+	keyboard = AddRowKeyboard(keyboard)
+
+	for _, qr := range msg.QuickReplies() {
+		keyboard = AddTextButton(qr, qr, "secondary", keyboard)
+	}
+	return keyboard
+}
+
+func AddRowKeyboard(keyboard Keyboard) Keyboard {
+	if len(keyboard.Buttons) == 0 {
+		keyboard.Buttons = make([][]buttonPayload, 1)
+	} else {
+		row := make([]buttonPayload, 0)
+		keyboard.Buttons = append(keyboard.Buttons, row)
+
+	}
+	keyboard.One_Time = true
+
+	return keyboard
+}
+
+func AddTextButton(label string, payload interface{}, color string, keyboard Keyboard) Keyboard {
+	b, err := json.Marshal(payload)
```

Comment:
I'm fine with not error checking JSON marshalling of things we control - but use `jsonx.MustMarshal`